pkg/volume/csi/csi_mounter_inline.go

```diff
@@ -0,0 +1,139 @@
+/*
+Copyright 2018 The Kubernetes Authors.
```

Comment:
It's not clear to me why we need this file. The inline stuff should be handled inside the normal codepaths for Attach/Detach in the ADC and Setup/TearDown in the Kubelet respectively. I believe the only difference should be where we get the information from